hamlet/celery.py

```diff
@@ -277,3 +278,97 @@ def workflow_export(
             raise e
     finally:
         os.unlink(out_f_name)
+
+
+# Creates a JSON file which details all the image URLs in a Subject Set, along
+# with that image's associated Subject information. JSON will be of the format:
+#
+# [ [ "http://image.url/example.png",
+#     "{ \"subject_info\": \"is a stringified JSON object\" }" ],
+#   ... ]
+#
+# To be used with the Zooniverse ML Subject Assistant web app (https://github.com/zooniverse/zoo-ml-subject-assistant),
+# and the associated (and currently unnamed) Microsoft machine learning service.
+# ------------------------------------------------------------------------------
```

Comment:
Could you move this comment into a docstring? (See here for an example https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)

Reply:
👌